package.json

```diff
@@ -38,13 +38,13 @@
         ]
     },
     "devDependencies": {
-        "@types/node": "18.17.15",
-        "@typescript-eslint/eslint-plugin": "^6.7.0",
-        "@vitest/coverage-v8": "^0.34.4",
-        "eslint": "8.49.0",
+        "@types/node": "20.7.0",
```

Comment:
Do we move it to node 20, or should we limit Dependabot to ignore this on version 20?

Reply:
@froblesmartin Let's move it to node20.